test/move.js

```diff
@@ -0,0 +1,56 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+describe('move', function() {
+  const list = ['a', 'b', 'c', 'd', 'e'];
+
+  it('should move an item to a higher idx', function() {
+    const expected = ['a', 'c', 'd', 'b', 'e'];
+    eq(RA.move(1, 3, list), expected);
+  });
+
+  it('should move an item to a lower idx', function() {
+    const expected = ['a', 'd', 'b', 'c', 'e'];
+    eq(RA.move(3, 1, list), expected);
+  });
+
```

Comment:
We're missing test for a case when `fromIdx` is out the list range.

Reply:
Well I think we should support the same as the `Array` supports.
```javascript
const list = ['a', 'b', 'c'];
list[-1]  = 'd'; list.length = 3;
list[100] = 'd'; list.length = 101;
```
...and so on. Does it make sense ?

Reply:
> Now I have noticed that ramda doesn't support negative indexes and the behaviour is undefined when they are supplied

Exactly. 

> we should only add test describing what happens if fromIdx is out the list range (undefined added to toIndex)

I'll add this now.

Reply:
@char0n That should be good to merge.

Reply:
Will do the final check tommorow